tests/test_confidence.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+from vulture import core
+
+dc = core.DEFAULT_CONFIDENCE
+
+
+def check_min_confidence(code, min_confidence, expected_dict={}, **expected):
```

Comment:
I'd rather just use `def check_min_confidence(code, min_confidence, expected):` then and always pass a dictionary.